litestar/_openapi/schema_generation/schema.py

```diff
@@ -298,8 +290,14 @@ def not_generating_examples(self) -> SchemaCreator:
         return type(self)(generate_examples=False, plugins=self.plugins, prefer_alias=False)
 
     def get_plugin_for(self, field_definition: FieldDefinition) -> OpenAPISchemaPluginProtocol | None:
+        def plugin_supports_field(plugin: OpenAPISchemaPluginProtocol, field: FieldDefinition) -> bool:
```

Comment:
Doesn't seem like this has to be an inline definition, so maybe this should just be a static method?

Reply:
I added it only to simplify the generator expression - I don't mind splitting it out though :+1: